spec/immutable-data-structures.html

```diff
@@ -6,27 +6,590 @@ <h1>Immutable Data Structures</h1>
+            1. Set _k_ to _k_ + 1.
+          1. Let _itemK_ be 0.
+          1. Repeat, while _itemK_ &lt; _itemCount_.
+            1. Let _E_ be _items_[_itemK_].
+            1. If Type(_E_) is Object, throw a *TypeError* exception.
+            1. Append _E_ to the end of _newList_.
+            1. Set _itemK_ to _itemK_ + 1.
+          1. Set _k_ to _actualStart_ + _actualDeleteCount_.
+          1. Repeat, while _k_ &lt; _len_,
+            1. Let _E_ be _list_[_k_].
+            1. Append _E_ to the end of _newList_.
+            1. Set _k_ to _k_ + 1.
+          1. Return a new Tuple value whose [[Sequence]] is _newList_.
+        </emu-alg>
+      </emu-clause>
+      <emu-clause id="sec-tuple.prototype.concat">
+        <h1>Tuple.prototype.concat ( ..._args_ )</h1>
+        <p>When the *concat* method is called with zero or more arguments, it returns a tuple containing the elements of the tuple followed by the elements of each argument in order.</p>
+        <p>The following steps are taken:</p>
+        <emu-alg>
+          1. Let _T_ be ? thisTupleValue(*this* value).
+          1. Let _list_ be a new empty List.
+          1. Let _n_ be 0.
+          1. Let _items_ be a List whose first element is _T_ and whose subsequent element are, in left to right order, the arguments that were passed to this function invocation.
+          1. Repeat, while _items_ is not empty,
+            1. Remove the first element from _items_ and let _E_ be the value of the element.
+            1. Let _spreadable_ be ? IsConcatSpreadable(_E_).
+            1. If _spreadable_ is *true*, then
+              1. Let _k_ be 0.
+              1. Let _len_ be ? LengthOfArrayLike(_E_).
+              1. If _n_ + _len_ &gt; 2<sup>53</sup> - 1, throw a *TypeError* exception.
+              1. Repeat, while _k_ &lt; _len_,
+                1. Let _P_ be ! ToString(_k_).
+                1. Let _exists_ be ? HasProperty(_E_, _P_).
+                1. If _exists_ is *true*, then
+                  1. Let _subElement_ be ? Get(_E_, _P_).
+                  1. If Type(_subElement_) is Object, throw a *TypeError* exception.
+                  1. Append _subElement_ to the end of list _list_.
+                1. Set _n_ to _n_ + 1.
+                1. Set _k_ to _k_ + 1.
+            1. Else,
+              1. NOTE: _E_ is added as a single item rather than spread.
+              1. If _n_ &ge; 2<sup>53</sup> - 1, throw a *TypeError* exception.
+              1. If Type(_E_) is Object, throw a *TypeError* exception.
+              1. Append _E_ to the end of list _list_.
+              1. Set _n_ to _n_ + 1.
+          1. Return a new Tuple value whose [[Sequence]] is _list_.
+        </emu-alg>
+      </emu-clause>
+      <emu-clause id="sec-tuple.prototype.includes">
+        <h1>Tuple.prototype.includes</h1>
+        <p>The initial value of the *Tuple.prototype.includes* is the *"includes"* property of the intrinsic object %Array.prototype%.</p>

```

Comment:
```suggestion
        <p>The initial value of *Tuple.prototype.includes* is %Array.prototype.includes%.</p>
```

Reply:
Why was this change made? The previous wording seems simpler.